naptan_update/update_table.py

```diff
@@ -38,7 +37,7 @@ def lambda_handler():
     # Connect to the DynamoDB table
     table = dynamo_db.Table(TABLE_NAME)
```

Comment:
Another comment, is about the line: `for line in my_zip_file.open(contained_file).readlines()` 

What his line is reads all of the lines into a list and then iterates over the list. This causes us both to iterate over all lines twice and uses a lot of memory - you can instead do: `for line in my_zip_file.open(contained_file)` - which iterates over all lines in the file.

but that isn't good enough either as you open a file handler and never close it a better way to go about it would be: 

```
with my_zip_file.open(contained_file) as f:
    for line in f:
        ...
```

Reply:
Reading more, I see in the end you only write it to another file line by line - this is a waste for a few reasons:
1. I am sure the zip library has a function to unzip a file to a path easier and faster
2. If you already open the file to iterate over it, let's do it just once and the things you do when you iterate below (update the stops) - Let's do here.